### ISSUE TITLE
[Mellanox] check_sysfs.py return empty result for psu_info["temp"]

### ISSUE THREAD

**Issue by bingwang-ms** [sonic-net/sonic-mgmt#10875]
**Description**
The issue is found in latest 202305 image on SN4600 testbed.
Incorrect `psu_info` is returned by `check_sysfs.py`.

```
 File "/azp/_work/40/s/tests/platform_tests/test_reboot.py", line 91, in reboot_and_check
    check_interfaces_and_services(dut, interfaces, xcvr_skip_list, reboot_type=reboot_type)
  File "/azp/_work/40/s/tests/platform_tests/test_reboot.py", line 140, in check_interfaces_and_services
    check_sysfs(dut)
  File "/azp/_work/40/s/tests/platform_tests/mellanox/check_sysfs.py", line 131, in check_sysfs
    psu_temp = float(psu_info["temp"]) / 1000
ValueError: could not convert string to float: ''

```
It's because `/var/run/hw-management/thermal/psu1_temp` doesn't exist on the device.
```
admin@str2-msn4600c-acs-03:~$ ls -al /var/run/hw-management/thermal/psu1_temp*
lrwxrwxrwx 1 root root 87 Nov 29 07:03 /var/run/hw-management/thermal/psu1_temp1 -> /sys/devices/platform/mlxplat/i2c_mlxcpld.1/i2c-1/i2c-4/4-0059/hwmon/hwmon4/temp1_input
lrwxrwxrwx 1 root root 85 Nov 29 07:03 /var/run/hw-management/thermal/psu1_temp1_max -> /sys/devices/platform/mlxplat/i2c_mlxcpld.1/i2c-1/i2c-4/4-0059/hwmon/hwmon4/temp1_max
lrwxrwxrwx 1 root root 87 Nov 29 07:03 /var/run/hw-management/thermal/psu1_temp2 -> /sys/devices/platform/mlxplat/i2c_mlxcpld.1/i2c-1/i2c-4/4-0059/hwmon/hwmon4/temp2_input
lrwxrwxrwx 1 root root 85 Nov 29 07:03 /var/run/hw-management/thermal/psu1_temp2_max -> /sys/devices/platform/mlxplat/i2c_mlxcpld.1/i2c-1/i2c-4/4-0059/hwmon/hwmon4/temp2_max
```

**Steps to reproduce the issue:**
1. Run any test case that calls `check_sysfs`. Such as `test_cold_reboot`, `test_check_hw_mgmt_sysfs`
2.
3.

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important:**

    **Output of `show version`:**

```
admin@str-msn4600c-acs-02:~$ show ver

SONiC Software Version: SONiC.20230531.10
SONiC OS Version: 11
Distribution: Debian 11.8
Kernel: 5.10.0-23-2-amd64
Build commit: bd152fdc10
Build date: Sun Nov 26 04:53:53 UTC 2023
Built by: cloudtest@af2b667cc000001
```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

**Comment by bingwang-ms** [comment]
@roy-sror Can you help take a look? Thanks!

**Comment by Junchao-Mellanox** [comment]
PR for this issue: https://github.com/sonic-net/sonic-mgmt/pull/10878

**Comment by bingwang-ms** [comment]
Fixed by #10878
